targets/targets.json

```diff
@@ -5270,6 +5270,61 @@
             "target.network-default-interface-type": "ETHERNET"
         }
     },
+    "ARM_MUSCA_A1": {
+        "public": false,
+        "inherits": ["Target"],
+        "supported_toolchains": ["ARMC6", "GCC_ARM"],
```

Comment:
still no IAR support , just checking the current status as I noticed it's not here .

Reply:
Yes, now that IAR 8 is in master we can do it.